docs/quickstart.rst

```diff
@@ -237,6 +257,36 @@ credentials::
 
 Starting Everything Up
 **********************
+
+Match Shotgun users with JIRA users (for JIRA servers hosted by Atlassian only)
+===============================================================================
+
+.. code-block:: bash
+
+    $ python update_shotgun_users.py --settings <path to your settings.py> --project <id of your project>
+
+.. note::
+    For every user found in Shotgun, the script will search for a JIRA user with
+    the same email address. Note that if you have multiple users in Shotgun with
```

Comment:
Suggestion:
"Note that if you have multiple users" > "If you have multiple users
As we already have it in a Note box.